src/app/zones/views/CreateView.vue

```diff
@@ -20,19 +21,15 @@
 
       <template #actions>
         <KButton
-          v-if="token === '' || isZoneConnected"
           appearance="outline"
           data-testid="exit-button"
-          :to="{ name: 'zone-cp-list-view' }"
-        >
-          {{ t('zones.form.exit') }}
-        </KButton>
-
-        <KButton
-          v-else
-          appearance="outline"
-          data-testid="exit-button"
-          @click="toggleConfirmModal"
+          @click="() => {
+            if(token === '' || isZoneConnected) {
+              route.replace({name: 'zone-cp-list-view'})
+            } else {
+              toggleConfirmModal()
+            }
+          }"
```

Comment:
**Issue**: Please don’t replace the proper link with a click-triggered route navigation. It’s a deliberate choice to have the thing that behaves like a link be a link.

Reply:
I actually think this should use `history.back`. There are two ways to get to this "modal" looking thing. One is from `/` and one is from `/zones`.

Having an 'Exit' button take me to a completely different page (i.e. if I clicked [Create] from `/`) is a little weird, it should take me to the page that I came from.

lmk

---

(Seeing as this is like a modal, normally the idea would be that I haven't actually left the URL that I'm on and the page at `/` should still be underneath the modal thing, this leads into a much larger discussion though, so for the moment we should probably just change this to use `history.back`)

Reply:
FYI I'm gonna look at moving this to use history.back instead, we can always change it back if we spot a reason why we shouldn't do this.

Reply:
I added `route.back` and used that so it now properly 'Exits' you back to where you originally came from. Resolving this, but feel free to reopen if necessary.

Reply:
Eh, I really don’t think this should use `history.back`. The last history entry could be anything including a non-Kuma GUI URL.

Can we move the changes to this element back, please? It doesn’t relate to this PR as far as I can tell.

Reply:
The last history entry is where you came from and therefore where you exit back to.

I'm not sure how many people will be adding links to their websites with a link to our zone creation page. If you come directly to it by entering the URL in your location bar it will default to the zone listing page instead. tbh I think even this case is incredibly rare, so I think catering for people how add links to the zone creation page to their own websites is even rarer, and depending where that link is, history.back is probably still the correct action for an `Exit` button

I'd be happy to change it back to how it was before you said this should be a link (and PR this separately), but I disagree that it needs to be a link and I think more than anything here, in this case I really don't think it matters too much, perhaps arguably if we are worried about accessibility we should make exit take you back to where you were in the most common case, not to potentially a completely different page.

Reply:
I went back to two buttons here, resolving, lmk if it needs reopening.

Reply:
See https://github.com/kumahq/kuma-gui/issues/1706